### ISSUE TITLE
Consider making `node-postal` an optional peer dependency.

### ISSUE THREAD

**Issue by wtgtybhertgeghgtwtg** [joaocarmo/i18n-postal-address#46]
### Expected Behavior

`i18n-postal-address` can be installed without any warnings, even if `node-postal` is not installed.  As per the README.md, the functionality is enables is experimental.  If it can be helped, I would prefer not to have a native dependency.

### Actual Behavior

For some package managers, adding a dependency without including its peer dependencies will give a warning.  With some configurations (e.g. PNPM where `strict-peer-dependencies` is set to `true`), it'll error out.

### Steps to Reproduce

1. Run `pnpm add i18n-postal-address` on a package that does not have `node-postal` as a dependency.
2. See warning.

### Your Environment

`pnpm@7.25.0`

I can file a PR for this, if you'd like.

**Comment by joaocarmo** [comment]
@wtgtybhertgeghgtwtg I've tried replicating the issue you mentioned, but I didn't encounter any problems. Here's the process I followed:

1. I installed `pnpm` using the command `brew install pnpm`.
2. I created a new repository with the commands `mkdir tmp`, followed by `cd tmp` and `pnpm init`.
3. I set the `strict-peer-dependencies` to `true` using the command `pnpm config set strict-peer-dependencies true`.
4. I then added the package using the command `pnpm add i18n-postal-address`.

```txt
$ pnpm config get strict-peer-dependencies
true
$ pnpm add i18n-postal-address
Packages: +5
+++++
Progress: resolved 5, reused 5, downloaded 0, added 0, done

dependencies:
+ i18n-postal-address 0.7.1

Done in 399ms
```

Let me know if there are any other specifics about your setup that might be causing the discrepancy.
